src/main/java/net/minestom/server/Viewable.java

```diff
@@ -58,40 +58,42 @@ default boolean isViewer(@NotNull Player player) {
      *
      * @param packet the packet to send to all viewers
      */
-    default void sendPacketToViewers(@NotNull SendablePacket packet) {
+    default void sendPacketToViewers(ServerSettingsProvider serverSettingsProvider, @NotNull SendablePacket packet) {
```

Comment:
I don't like that ServerSettings is required here, but I don't know how to fix it